src/components/userFeedback.js

```diff
@@ -0,0 +1,113 @@
+import '../styles/components/userFeedback.scss'
+
+import React, { useRef, useState } from 'react'
+import HubspotForm from 'react-hubspot-form'
+import Loading from '../components/loading'
+import { FormattedMessage } from 'react-intl'
+import { trackCustomEvent } from 'gatsby-plugin-google-analytics'
+
+const UserFeedback = ({ title, locale }) => {
+  const feedbackBodyRef = useRef(null)
+  const feedbackCloseRef = useRef(null)
+  const [showFollowUp, setShowFollowUp] = useState(false)
+  const [showYesFollowUp, setShowYesFollowUp] = useState(false)
+
+  const setDocHelpful = (docTitle, isHelpful) => {
```

Comment:
It's better to write `const setDocHelpful = (docTitle, isHelpful) => () => {` to prevent redundant func re-created.